src/nvim/ex_docmd.c

```diff
@@ -2093,8 +2114,9 @@ int parse_command_modifiers(exarg_T *eap, char_u **errormsg, bool skip_only)
       cmdmod.split |= WSP_BOT;
       continue;
 
-    case 'c':   if (!checkforcmd(&eap->cmd, "confirm", 4))
+    case 'c':   if (!checkforcmd(&eap->cmd, "confirm", 4)) {
         break;
+    }
```

Comment:
this one a little weird. probably uncrustify failed to consider `if (expr) {` on the same line as `case 'c'` (and the fix is to move the `if` to the line below the `case` ).

Reply:
there's a few more cases like this, could be fixed manually with a suitable pattern for now :)

Reply:
It's possible to add a newline after a case. Want me to try it out and add it as a separate commit so you can check if it looks good?

Reply:
yes.

Reply:
Added newline after case, but only if there's no brace after.

Reply:
While you're at it, could you take an extra look at the indentation on switch/case? I think there's currently no indentation between switch and case, is that correct?

Reply:
> currently no indentation between switch and case, is that correct?

correct, though I don't think it matters much. Except that the current rule avoids some gratuitous indentation.